#This will contain an image-pair (matching), will be published with edge message for analysis
time c_timestamp
time prev_timestamp
float32 goodness
sensor_msgs/Image curr_image
sensor_msgs/Image prev_image
string curr_label
string prev_label
